third_party/2and3/redis/client.pyi

```diff
@@ -65,9 +65,14 @@ class Redis(object):
         ssl: bool = ...,
         ssl_keyfile: Optional[Text] = ...,
         ssl_certfile: Optional[Text] = ...,
-        ssl_cert_reqs: Optional[Union[str, int]] = ...,
+        ssl_cert_reqs: Union[str, int] = ...,
```

Comment:
This was `Optional` before, and looking at the code of `SSLConnection`, there is an explicit `None` check, so the old version seems to be correct.

```suggestion
        ssl_cert_reqs: Optional[Union[str, int]] = ...,
```

Reply:
Ah, I missed the None check. I've reverted to `Optional` here